language-server/modules/langserver-core/src/test/resources/completion/expression_context/source/table_constructor_expr_ctx_source5.bal

```diff
@@ -0,0 +1,7 @@
+type Employee1 record {
+    readonly string firstname;
+    readonly string lastname;
+    int salary;
+};
+
+table<Employee1> key <string> t4 = table key()
```

Comment:
Shall we add another test with a key constraint of record type?

Reply:
Done